Admin/AbstractAdmin.php

```diff
@@ -2382,6 +2398,13 @@ public function setBaseCodeRoute($baseCodeRoute)
      */
     public function getBaseCodeRoute()
     {
+        if ($this->isChild()) {
+            return $this->getParent()->getBaseCodeRoute().'|'.$this->getCode();
+        }
```

Comment:
Hum so to sum up your changes, the code is no longer computed when computable, but when needed, just in time.